tests/sentry/api/endpoints/test_api_token_details.py

```diff
@@ -0,0 +1,154 @@
+from rest_framework import status
+
+from sentry.models.apitoken import ApiToken
+from sentry.testutils.cases import APITestCase
+from sentry.testutils.silo import control_silo_test
+
+
+@control_silo_test
+class ApiTokenDetailTest(APITestCase):
```

Comment:
Maybe we can name the tests based on the methods theyre testing?

so this would be `ApiTokenGetTest`
then PUT
then DELETE